server/src/models/drug.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from 'mongoose';
+
+const drugSchema = new mongoose.Schema({
+  activeIngredient: {
+    type: String,
+    required: true
+  },
+  name: {
+    type: String,
+    required: true
+  },
+  description: String,
+  applicationRequirement: mongoose.SchemaTypes.ObjectId,
```

Comment:
Marcin zerknij tutaj proszę, czy ten typ jest odpowiedni?

Reply:
Narazie jest git, jak pojawia się model dla tych obiektów to się zmieni